src/cgame/cg_key_name.cpp

```diff
@@ -59,7 +59,7 @@ static int bindTeam;
 void CG_SetBindTeam(team_t team) {
     switch (team) {
     case TEAM_NONE:
-        bindTeam = 3; // BIND_TEAM_SPECTATORS
+        bindTeam = NUM_TEAMS; // BIND_TEAM_SPECTATORS
```

Comment:
Remove unrelated change

Reply:
It was part of the original change by freem. I think he wanted to use a constant rather than a naked number.